### PR TITLE
add functionality to check response length

### PR DESCRIPTION
**Pull request by xsuoownit**

### CODE REVIEW

File: src/main/java/com/github/cchacin/cucumber/steps/RestSteps.java [hunk 1]
```diff
@@ -178,4 +179,10 @@ public void response_json_path_element_should_be(final String jsonPath, final St
         final Object responseValue = JsonPath.read(this.responseValue, jsonPath);
         assertThat(String.valueOf(responseValue)).isEqualTo(value);
     }
+
+    @Then("^response json path list \"(.*?)\" should be of length (.*?)$")
```
Comment (cchacin) [inline src/main/java/com/github/cchacin/cucumber/steps/RestSteps.java]: Can you check the regular expression for digits?

Comment (xsuoownit) [inline src/main/java/com/github/cchacin/cucumber/steps/RestSteps.java]: use regular expression for digits and updated version https://github.com/cchacin/cucumber-common-steps/commit/66f9b2ddd13fc1fb324d837ff776bbeca512172c

**Comment by cchacin** [comment]
LGTM

File: src/main/java/com/github/cchacin/cucumber/steps/RestSteps.java (hunk 1 above)
Comment (cchacin) [inline src/main/java/com/github/cchacin/cucumber/steps/RestSteps.java]: Version 0.2.2 released

**Comment by cchacin** [comment]
@xsuoownit v0.2.2 released

**Comment by xsuoownit** [comment]
@cchacin thanks, I will make the changes in search

**Pull request merged**